unzip_http.py

```diff
@@ -129,7 +129,10 @@ def infoiter(self):
             warning(f"{hostname} Accept-Ranges header ('{r}') is not 'bytes'--trying anyway")
 
         self.zip_size = int(resp.headers['Content-Length'])
-        resp = self.get_range(self.zip_size-65536, 65536)
+        resp = self.get_range(
+            self.zip_size-65536 if self.zip_size > 65536 else 0,
```

Comment:
Hey @horw, how about `max(0, ...)`?  and similarly below.

Reply:
You are right